webClient/src/app/editor/frame/frame.component.scss

```diff
@@ -162,7 +162,7 @@
       }
     }
     .gz-editor {
-      // width: 100%;
+      width: 600px;

```

Comment:
Rest is OK, but this one doesn't make sense. What would happen if I had the window smaller or larger?

Reply:
To be quite honest I am not sure why this fixed the issue, however its behavior is consistent between Chrome and Firefox.  
Here I have expanded the window past 600px, and the fitment works perfectly even with the fixed width.  It seems that setting the width to anything expect for 100% fixes the issue of the view-lines expanding past their intended container.
![image](https://user-images.githubusercontent.com/7998484/59867187-87227480-935b-11e9-92b6-1378285119b3.png)

Reply:
@1000TurquoisePogs

Reply:
This is not an issue that was included as part of the ticket, but it occurs on the version of zlux-editor in the staging branch of :zowe/zlux-editor as well